### ISSUE TITLE
Undistort and Transform Perspective Quiz

### ISSUE THREAD

**Issue by Deborah-Digges** [udacity/sdc-issue-reports#364]
When I submit the solution to this Quiz, I get the message "Oops! looks like you got a different transform matrix than I was expecting..."

However, my undistortion and perspective transform looks correct:

<img width="567" alt="screen shot 2017-01-14 at 7 01 50 pm" src="https://cloud.githubusercontent.com/assets/3866405/21955242/c76617a2-da8c-11e6-9574-04e731f27f91.png">

My intuition tells me that the transform matrix will differ from person to person depending on the shape of the rectangle they choose. Isn't that right?

**Comment by Deborah-Digges** [comment]
I chose the same source points as the solution, but different "destination points".

If I choose the same destination points as the solution, I pass the test. However, I'm wondering if this is right. 

I dynamically computed a rectangle which approximately bounds the original rectangle area and then applied the transform.

Is it necessary to define the destination points to be the boundaries of the image?

**Comment by ryan-keenan** [comment]
Thanks for reporting this Deborah!  You're right that different choice of points will yield different results.  In this quiz we did build in a margin of tolerance for the transform matrix `M`, but we're asking you to shoot for output that looks approximately like the example image shown above the quiz.  The tolerance was probably too tight, so I've opened it up a bit.  I've also added a comment to the quiz asking for output that looks like the example.  Hopefully this helps!
